test/e2e/apps/statefulset.go

```diff
@@ -843,6 +847,75 @@ var _ = SIGDescribe("StatefulSet", func() {
 				return nil
 			}, framework.StatefulPodTimeout, 2*time.Second).Should(BeNil())
 		})
+
+		It("should have a working scale subresource", func() {
```

Comment:
@liggitt I am not sure whether i catch your meaning. This e2e test does not work yet as the scale subresource cannot be found.

What i want is to obtain a JSON blob located at:
(1) for StatefulSet: e.g., `http://localhost:8001/apis/apps/v1beta2/namespaces/default/statefulsets/`
(2) for StatefulSet's scale subresource: e.g., `http://localhost:8001/apis/apps/v1beta2/namespaces/default/statefulsets/web/scale`

I am able to test it manually using `kubectl proxy` and `curl`. The subresource `Spec.Replicas` attribute can be modified, and the change will be reflected to the StatefulSet `Spec.Replicas` attribute.

I am not sure whether `RESTClient` is the suitable tool for this. I will look more into the library, but do you have any recommended person that i can ask for help with regarding this? Thanks!